drivers/azure/azureutil/azureutil.go

```diff
@@ -564,7 +571,7 @@ func (a AzureClient) CreateVirtualMachine(resourceGroup, name, location, size, a
 						Sku:       to.StringPtr(img.sku),
 						Version:   to.StringPtr(img.version),
 					},
-					OsDisk: getOSDisk(name, osDiskBlobURL, isManaged, storageType, diskSize),
+					OsDisk: getOSDisk(name, storageAccount, isManaged, storageType, diskSize),
```

Comment:
It doesn't look like osDiskBlobURL is used anywhere else but it's still being created above